tools/build_chain.sh

```diff
@@ -1514,10 +1516,10 @@ download_bin()
     if [ $(curl -IL -o /dev/null -s -w %{http_code}  ${cdn_link_header}/v${compatibility_version}/${package_name}) == 200 ];then
         curl -LO ${Download_Link} --speed-time 20 --speed-limit 102400 -m ${download_timeout} || {
             LOG_INFO "Download speed is too low, try ${cdn_link_header}/v${compatibility_version}/${package_name}"
-            curl -LO ${cdn_link_header}/v${compatibility_version}/${package_name}
+            curl -#LO ${cdn_link_header}/v${compatibility_version}/${package_name}
         }
     else
-        curl -LO ${Download_Link}
+        curl -#LO ${Download_Link}
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5878472)